### PR TITLE
Clean container logs

### PR DESCRIPTION
**Pull request by xbglowx**
@discordianfish not sure if I introduced a docker-proper.go bug when I was updating it to work with the latest dockerclient (never touched go before), but now the container fails to stay running if it can't delete a image, due to being used by a container.

```
3549 ➜  ~/vcs/docker/docker-proper git:(clean_container_logs) docker run -d --name docker-proper -v /var/run/docker.sock:/docker.sock -v /var/lib/docker/containers/:/mnt/docker/containers  docker-proper -a unix:///docker.sock -ia 1h -ca 1h -r 24h
7e964870a37eb0987aacd83a4cbf62a559559b23f325bd7af771da13a819ba56

3551 ➜  ~/vcs/docker/docker-proper git:(clean_container_logs) ✗ docker logs docker-proper
2015/05/08 03:40:08 Found 0 expired containers
2015/05/08 03:40:08 Found 1 expired images
2015/05/08 03:40:08 rmi 41b730702607edf9b07c6098f0b704ff59c5d4361245e468c0d551f50eae6f84
2015/05/08 03:40:08 Couldn't remove image: 409 Conflict: Conflict, cannot delete 41b730702607 because the running container 7e964870a37e is using it, stop it and use -f to force

3552 ➜  ~/vcs/docker/docker-proper git:(clean_container_logs) ✗ docker ps | grep docker-proper
3553 ➜  ~/vcs/docker/docker-proper git:(clean_container_logs) ✗
```

If I change the time thresholds, so it doesn't find any candidates, it continues to run

```
3554 ➜  ~/vcs/docker/docker-proper git:(clean_container_logs) ✗ docker run -d --name docker-proper -v /var/run/docker.sock:/docker.sock -v /var/lib/docker/containers/:/mnt/docker/containers  docker-proper -a unix:///docker.sock -ia 1000h -ca 1000h -r 24h
3f6d0cacef1463adcf2516f1a02eedd73ea1591c05e5e2535a9417abe426a9fa

3555 ➜  ~/vcs/docker/docker-proper git:(clean_container_logs) ✗ docker logs docker-proper
2015/05/08 03:42:29 Found 0 expired containers
2015/05/08 03:42:29 Found 0 expired images
2015/05/08 03:42:29 Sleeping for 24h0m0s

3556 ➜  ~/vcs/docker/docker-proper git:(clean_container_logs) ✗ docker ps | grep docker-proper
3f6d0cacef14        docker-proper:latest          "./docker-proper -a    5 seconds ago       Up 4 seconds                             docker-proper
```

Can you please have a look?

### CODE REVIEW

**Comment by xbglowx** [comment]
I don't think I introduced a bug, since the current version (untouched by me) is failing to start

```
ops@docker018:~$ docker ps | grep 'docker-proper'
16e26b52c27b        dckr/docker-proper:latest        "./docker-proper -a    4 months ago        Restarting (1) 2 days ago                                                                                     docker-proper                                                                                     docker-proper
```

File: proper/docker-proper.go [hunk 1]
```diff
@@ -147,7 +147,8 @@ func removeImages(client *dockerclient.DockerClient, images []*dockerclient.Imag
 	for _, image := range images {
 		log.Print("rmi ", image.Id)
 		if !*dry {
-			if err := client.RemoveImage(image.Id); err != nil {
+			_, err := client.RemoveImage(image.Id)
+			if err != nil {
 				/*				if derr, ok := err.(dockerclient.Error); ok {
```
Comment (discordianfish) [inline proper/docker-proper.go:152]: I guess we need to include this again so it skips images that are still running?

**Comment by discordianfish** [comment]
Why the path change?
In general I wouldn't add the log cleaning stuff. Especially since this is a public project and people won't expect that this just deletes their logs ;)
As discussed, I would clean up logs manually for now and make sure in the future the logs are shipped with heka.

**Comment by xbglowx** [comment]
Sorry, forgot this was public and I don't want to confuse others.

I figured, why do things manually, when I can automate and be done with it. Please ignore this diff.

**Pull request closed without merging**